drf_instamojo/signals/handlers.py

```diff
@@ -37,7 +37,12 @@ def payment_record_handler(instance: Payment, sender, **kwargs):
     if pr_status.get("success"):
         payment_request_imojo = pr_status.get("payment_request")
 
-        if payment_request_imojo.get("modified_at") > pr.modified_at:
+        # Hacky way to fix TypeError here.
```

Comment:
Why is this a hackway? I think this is a proper way

Reply:
Well, thanks for approving. I'll update my comment next time.